xbmc/addons/kodi-addon-dev-kit/include/kodi/StreamCodec.h

```diff
@@ -18,5 +18,9 @@ enum STREAMCODEC_PROFILE
   H264CodecProfileHigh,
   H264CodecProfileHigh10,
   H264CodecProfileHigh422,
-  H264CodecProfileHigh444Predictive
+  H264CodecProfileHigh444Predictive,
+  VP9CodecProfile0 = 20,
+  VP9CodecProfile1,
+  VP9CodecProfile2,
+  VP9CodecProfile3
```

Comment:
add a comma at the end, so the line doesn't have to be change if an additional value is added later